src/ComponentBase.ts

```diff
@@ -271,35 +278,35 @@ abstract class ComponentBase<P extends React.Props<any>, S extends Object> exten
         return !subscription.used;
     }
 
-    private _onSubscriptionChanged(subscription: StoreSubscription<S>, changedItem: any) {
+    private _onSubscriptionChanged<K extends keyof S>(subscription: StoreSubscription<S>, changedItem: any) {
```

Comment:
Remove the type parameter. The only caller does not specific the type parameter, so it is inferred as some default (e.g. `string` or `any`). That is objectively useless.